src/inputs.ts

```diff
@@ -22,12 +22,14 @@ import {
 
 // Inputs are the inputs to the action, matching the inputs in the action.yml.
 export interface Inputs {
+  ref_name: string;
```

Comment:
what is this and why is it here? this is not a named option in action.yml

Reply:
Its resolved from the inputs, not a parameter to the action.